openmdao/recorders/sqlite_recorder.py

```diff
@@ -687,3 +701,19 @@ def shutdown(self):
         # close database connection
         if self.connection:
             self.connection.close()
+
+    def delete_recordings(self):
+        """
```

Comment:
Is this function part of another story? it never gets used and isn't tested (and the comment is cut-and-paste.)

Reply:
I use it four times in test_sqlite_recorder.py. I use it to clear out the case recorder file so I can do a lot of runs with different options and not have to make a whole new test. 

Good catch on the comment! Deleted.

Reply:
I see, i must have had that file closed when I searched.  It makes sense now.